core/templates/dev/head/services/context.service.spec.ts

```diff
@@ -150,41 +156,6 @@ describe('Context service', function() {
     });
   });
 
-  describe('behavior in the question editor view', function() {
```

Comment:
Why are these tests deleted? Shouldn't they be replaced somewhere?

Reply:
I deleted the functions that they tested, since we don't have a question_editor view.

Reply:
Done, added tests that tests the question editor modal.